src/files.ts

```diff
@@ -247,8 +247,8 @@ export async function* collectBundleFiles({
     // Check if symlink and exclude if requested
     if (!fileStats || (fileStats.isSymbolicLink() && !symlinksEnabled)) continue;
 
-    if (fileStats.isFile() && fileStats.size <= MAX_FILE_SIZE) {
-      files.push(path);
+    if (fileStats.isFile()) {
+      fileStats.size <= MAX_FILE_SIZE ? files.push(path) : yield path;
```

Comment:
question: you sure this doesn't break anything? Previously we would continue even if file would be bigger than the limit and now we are returning here.

Reply:
I don't think it would break since what happens after this is the sanitisation of these files, which basically gets the `FileInfo` from the files. This is necessary for the files that need to be scanned.
However, since I don't really need the `FileInfo` from the files that are bigger than the max file size, I can just return its path here and there's no need for sanitising.

So the only thing that this is adding is another 'pause' to the generator for the files that exceed the max size, that translates in having more values returned from the function in `bundles.ts` on the for cycle that goes through the collector, but it doesn't necessarily break anything.

Reply:
Thank you for explanation, I understand.